ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh

```diff
@@ -24,14 +24,24 @@ if [[ "${CLUSTER_TYPE:-}" =~ ^aws-s?c2s$ ]]; then
   REGION=$(jq -r ".\"${LEASED_RESOURCE}\".source_region" "${CLUSTER_PROFILE_DIR}/shift_project_setting.json")
 fi
 
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
 # 1. get vpc id and public subnet
-VpcId=$(cat "${SHARED_DIR}/vpc_id")
+if [[ ! -f "${SHARED_DIR}/vpc_id" && ! -f "${SHARED_DIR}/public_subnet_ids" ]]; then
+  if [[ -e ${SHARED_DIR}/metadata.json ]]; then
```

Comment:
How about removing the check of metadata.json, so that the step can failed earlier.

Reply:
good idea, created https://github.com/openshift/release/pull/43314 for this